framework/messages/hr/yii.php

```diff
@@ -0,0 +1,105 @@
+    'No' => 'Ne',
+    'No help for unknown command "{command}".' => 'Nema pomoći za nepoznatu naredbu "{command}"',
+    'No help for unknown sub-command "{command}".' => 'Nema pomoći za nepoznatu pod-naredbu "{command}"',
+    'No results found.' => 'Nema rezultata.',
+    'Only files with these MIME types are allowed: {mimeTypes}.' => 'Samo datoteke s ovim MIME vrstama su dopuštene: {mimeTypes}.',
+    'Only files with these extensions are allowed: {extensions}.' => 'Samo datoteke s ovim eksentinzijama su dopuštene:: {extensions}',
+    'Page not found.' => 'Stranica nije pronađena.',
+    'Please fix the following errors:' => 'Molimo vas ispravite pogreške:',
+    'Please upload a file.' => 'Molimo vas da uploadate datoteku.',
+    'Showing <b>{begin, number}-{end, number}</b> of <b>{totalCount, number}</b> {totalCount, plural, one{item} other{items}}.' => 'Prikazuj  <b>{begin, number}-{end, number}</b> od <b>{totalCount, number}</b> {totalCount, plural, one{red} other{redova}}.',
+    'The file "{file}" is not an image.' => 'Podatak "{file}" nije slika.',
+    'The file "{file}" is too big. Its size cannot exceed {limit, number} {limit, plural, one{byte} other{bytes}}.' => 'Podatak "{file}" je prevelik. Ne smije biti veći od {limit, number} {limit, plural, one{bajt} other{bajtova}}.',
+    'The file "{file}" is too small. Its size cannot be smaller than {limit, number} {limit, plural, one{byte} other{bytes}}.' => 'Podatak "{file}" je premalen. Ne smije biti manji od {limit, number} {limit, plural, one{bajt} other{bajtova}}.',
+    'The format of {attribute} is invalid.' => 'Format od {attribute} je nevažeći.',
+    'The image "{file}" is too large. The height cannot be larger than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'Slika "{file}" je prevelika. Visina slike ne smije biti veća od {limit, number} {limit, plural, one{piksel} other{piksela}}.',
+    'The image "{file}" is too large. The width cannot be larger than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'Slika "{file}" je prevelika. Širina slike ne smije biti veća od {limit, number} {limit, plural, one{piksel} other{piksela}}.',
+    'The image "{file}" is too small. The height cannot be smaller than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'Slika  "{file}" je premalena. Visina slike ne smije biti manja od {limit, number} {limit, plural, one{piksel} other{piksela}}.',
+    'The image "{file}" is too small. The width cannot be smaller than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'Slika "{file}" je premalena.  Širina slike ne smije biti manja od {limit, number} {limit, plural, one{piksel} other{piksela}}.',
+    'The verification code is incorrect.' => 'Kod za provjeru nije točan.',
+    'Total <b>{count, number}</b> {count, plural, one{item} other{items}}.' => 'Ukupno <b>{count, number}</b> {count, plural, one{red} other{redova}}.',
```

Comment:
Same thing I said about the "Showing <b>{begin, number}-{end, number}</b> of <b>{totalCount, number}</b> {totalCount, plural, one{item} other{items}}." translation. 

This string is used in list views too, where there are items, not lines.